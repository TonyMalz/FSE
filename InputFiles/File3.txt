1, ,I
2,1,U
3,2,1,2
4,2,3,4
5,1,1,4
