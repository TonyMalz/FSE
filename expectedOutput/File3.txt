1 : I -> {1, 4}
|
+- 2 : U -> {1, 2, 3, 4}
|  |
   +- 3 : {1, 2} -> {1, 2}
|  |
   +- 4 : {3, 4} -> {3, 4}
|
+- 5 : {1, 4} -> {1, 4}